core/src/main/scala/dagr/core/tasksystem/Pipe.scala

```diff
@@ -74,6 +74,9 @@ trait PipeOut[Out] extends Pipe[Nothing,Out]
 // Section: Objects and private utility classes to make it all work
 ///////////////////////////////////////////////////////////////////////////////
 object Pipes {
+  /** The string that is sandwiched between commands to form pipes. */
+  val PipeString = "\\\n    | "
```

Comment:
The thing is, I want the `\n` evaluated, but I also want a backslash to escape the end of line.